src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,61 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplate(versionStr: string, studio: string, from: string, to: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplate.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[]
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[]
+
+			gfxTemplates.filter(g => g.IsDesign).map(g => designTemplates.push(g))
+
+			context.setBaseConfig(to, (designTemplates as unknown) as ConfigItemValue)
+		}
+	})
+}
+
+export function removeDesignChangesFromGFXTemplate(versionStr: string, studio: string, configId: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.removeDesignChangesFromGFXTemplate.${configId}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(configId) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(configId) as unknown) as TableConfigItemGFXTemplates[]
+
+			const newGFXTemplate = gfxTemplates.filter(g => !g.IsDesign)
```

Comment:
Minor thing `newGFXTemplate` is an array of TableConfigItemGFXTemplates, so the name should be plural to indicate this.
For extra niceness you could include the type declaration of the variable as well :)

Reply:
Changed in [91af7ee](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/91af7eea234989f636428c9e8cd8bd90a524675a)